Import guides (#42)
fivetran/terraform-provider-fivetran: docs/resources/connector.md

- `code` 
- `message` 

## Import

To import an existing `fivetran_connector` resource into your Terraform state, you need to get **Fivetran Connector ID** on the **Setup** tab of the connector page in your Fivetran dashboard.

1. Retrieve all connectors in a particular group using the [fivetran_group_connectors data source](/docs/data-sources/group_connectors). To retrieve existing groups, use the [fivetran_groups data source](/docs/data-sources/groups).

> 2. Define an empty resource in your `.tf` configuration:

Reviewer: ```suggestion
3. Define an empty resource in your `.tf` configuration:
```

```hcl
resource "fivetran_connector" "my_imported_connector" {

}
```

3. Call the `terraform import` command:

```
terraform import fivetran_connector.my_imported_connector <your Fivetran Connector ID>
```

> 4. Copy and paste the connector properties from the state to your `.tf` configuration using the `terraform state show` command:

Reviewer: ```suggestion
5. Copy and paste the connector properties from the state to your `.tf` configuration using the `terraform state show` command:
```

> 5. Copy and paste the connector properties from the state to your `.tf` configuration using the `terraform state show` command:

Reviewer: ```suggestion
5.  Using the `terraform state show` command to get the values from your state:
```

```
terraform state show 'fivetran_connector.my_imported_connector'
```

> 

Reviewer: ```suggestion
6. Copy the values from the state and paste them to your `.tf` configuration.
```

> 6. Copy the values from the state and paste them to your `.tf` configuration.

Reviewer: ```suggestion
6. Copy the values from the state and paste them to your `.tf` configuration.

```

> -> The `config` object in the state contains all properties defined in the schema. You need to remove properties from the `config` that are not related to connectors. See [Fivetran REST API documentation](https://fivetran.com/docs/rest-api/connectors/config) for reference to find the properties you need to keep in the `config` section.

Reviewer: ```suggestion
-> The `config` object in the state contains all properties defined in the schema. You need to remove properties from the `config` that are not related to connectors. See the [Fivetran REST API documentation](https://fivetran.com/docs/rest-api/connectors/config) for reference to find the properties you need to keep in the `config` section.
```
